### ISSUE TITLE
random sequence animation

### ISSUE THREAD

**Issue by VonHirsch** [ManiacalLabs/BiblioPixel#1149]
Something like:

```
animation:
    typename: sequence
    length: 2
    random: true
    animations:
        - $bpa.matrix.Mainframe
        - $bpa.matrix.MatrixRain

shape: [8, 8]
```

Gives: `TypeError: '_AnimationList' object does not support item assignment`

Is that to be expected?  Sorry for opening issues :(

**Comment by adammhaile** [comment]
@VonHirsch - email me (adammhaile [at] gmail)

**Comment by estiens** [comment]
I think you just have a yml spacing error?

```
animation:
  typename: sequence
  length: 600
  animations:
  - fireflies_remote.FireFliesRemote
```

**Comment by VonHirsch** [comment]
possibly, but I think I tried a few times, even with json.

add `random: true` underneath `typename: sequence` and see if it works for you.

**Comment by adammhaile** [comment]
fixed by #1150
